src/scripts/components/stories/pilot-story/components/chapter-progress-indication/chapter-progress-indication.tsx

```diff
@@ -87,12 +137,20 @@ const ChapterProgressIndication: FunctionComponent<Props> = ({
         className,
         progressIndicationElement
       )}
+      data-is-subchapter={isSubChapter}
       style={style}>
-      {Array.from({length: chapters.length}).map((_, index) => (
+      {Array.from({length: length > 1 ? length : 2}).map((_, index) => (
         <span
           key={index}
           data-is-selected={index === selectedChapterIndex}
-          onClick={() => scrollToChapterIndex(index)}
+          onClick={() =>
+            // Todo: Refactor
+            // 1. Get rid of nested turning
+            // 2. Only works because there is currently just one subchapter
+            // 3. Code is duplicated from nav-chapter-overview.ts
+            // eslint-disable-next-line no-nested-ternary
+            scrollToChapterIndex(isSubChapter ? (index === 0 ? 5 : 6) : index)
+          }
```

Comment:
Can you maybe move the code inside the `if/else` conditions into named functions? This would help to get an impression of what is happening here.

Reply:
Not sure what you mean 😅

Reply:
The component `ChapterProgressIndication` is quite big. I meant to move some of the code into functions to increase readability.

Reply:
done. I had the function already, wasn't using it ...